src/api/RequestId.ts

```diff
@@ -53,11 +53,11 @@ export class RequestId {
   }
 
   /**
-   * Converts the RequestId to a bigint.
-   * @returns The bigint representation of the request ID.
+   * Converts the RequestId to a BitString.
+   * @return The BitString representation of the RequestId.
    */
-  public toBigInt(): bigint {
-    return BigInt(`0x01${HexConverter.encode(this.hash.imprint)}`);
+  public toBitString(): BitString {
+    return new BitString(this.hash.imprint);
```

Comment:
let's also have BitString.fromDataHash(DataHash) static method or we'll confuse `this.hash.imprint` vs `this.hash.data`